src/mesh/side_impl.hh

```diff
@@ -99,7 +59,32 @@ inline Side::Side(const Mesh * mesh, unsigned int elem_idx, unsigned int set_lnu
     }
 
 
-    //inline void *Side::make_ptr() const {
-    //    return (void *)((long int) element() << (2 + side_idx_) );
-    //}
+
+///////////////////////////
+// SIDE_ITER inline implementation
+///////////////////////////
+
```

Comment:
Side is just an accessor anyway. historical purpose of SideIter was to simplify transition from pointers (Side *).
If you implement == and ++(prefix) in Side, we can use generic iterator:
Use tools/general_iterator:Iter as
like:
Iter<Side>